### ISSUE TITLE
 Modelangelo eval_per_resid | Attribute Error

### ISSUE THREAD

**Issue by jacob-r-anderson** [3dem/model-angelo#36]
On Ubuntu 20.04 and CentOS I have reproducibly gotten an attribute error when running ModelAngelo's `eval_per_resid` option. On both devices if I run the following, I get the same error.

```
model_angelo eval_per_resid --predicted-structure run1.cif --target-structure myfile.cif  --output-file out.txt


Traceback (most recent call last):
  File "/programs/x86_64-linux//modelangelo/0.2.2/bin/model_angelo", line 33, in <module>
    sys.exit(load_entry_point('model-angelo==0.2.2', 'console_scripts', 'model_angelo')())
  File "/programs/x86_64-linux/modelangelo/0.2.2/miniconda3/lib/python3.9/site-packages/model_angelo/__main__.py", line 51, in main
    args.func(args)
  File "/programs/x86_64-linux/modelangelo/0.2.2/miniconda3/lib/python3.9/site-packages/model_angelo/apps/eval_per_resid.py", line 286, in main
    ) = get_residue_fit_report(
  File "/programs/x86_64-linux/modelangelo/0.2.2/miniconda3/lib/python3.9/site-packages/model_angelo/apps/eval_per_resid.py", line 36, in get_residue_fit_report
    target_correspondence.astype(np.int32),
AttributeError: 'list' object has no attribute 'astype'
```

I have tried changing the "cif" files and the output file (directory, filename, and extension).

**Comment by jamaliki** [comment]
Hi,

Thank you for the bug request, I will push a hot-fix shortly.

Best,
Kiarash.

**Comment by jamaliki** [comment]
This is fixed as of release v0.2.3
